docs/source/examples/distributed-jobs.rst

```diff
@@ -47,3 +48,5 @@ SkyPilot exposes two environment variables to distinguish per-node commands:
   You can retrieve the number of nodes by :code:`echo "$SKY_NODE_IPS" | wc -l`
   and the IP address of the third node by :code:`echo "$SKY_NODE_IPS" | sed -n
   3p`.
+- :code:`SKY_NUM_GPUS_PER_NODE`: number of GPUs reserved on each node (the same
```

Comment:
L41: `SkyPilot exposes these environment variables that can be accessed in a task's ``run`` commands:`